static/js/main.js

```diff
@@ -13505,3 +13505,18 @@ $(document).ready(function() {
 $(document).ready(function() {
     $('.alert').createAlert();
 });
+
+/* ==========================================================================
+   Footer Button: Scroll to Top
+   ========================================================================== */
+
+$(document).ready(function() {
+    var duration = 300;
+
+    $('.footer_top-button').click(function(event) {
```

Comment:
Should we consider breaking apart styling and scripting hooks with namespacing? http://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/